src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const invertedRobot = {};
+
+  for (const key in robot) {
+    if (robot[key] in invertedRobot) {
+      return null;
+    }
+
+    invertedRobot[robot[key]] = key;
```

Comment:
When assigning the key to the invertedRobot, it's safer to ensure that the key is a string. Since object keys are always strings, if the value is not a string, it could lead to unexpected behavior. Use 'String(key)' to convert it explicitly.